lib/ProductOpener/Ingredients.pm

```diff
@@ -389,9 +395,13 @@ sub extract_ingredients_from_text($) {
 	return if not defined $product_ref->{ingredients_text};
 
 	my $text = $product_ref->{ingredients_text};
-	$text =~ s/\&quot;/"/g;
+	my $lc = $product_ref->{lc};
```

Comment:
This hides `$ProductOpener::Lang::lc`, doesn't it? Could be a bit confusing.

Reply:
I removed that one. There are other instances of $lc hiding in Tags.pm especially, but then I think the right thing will be to rename $ProductOpener::Lang::lc to $ProductOpener::Lang::site_lc 
It's a much bigger change though, let's do it separately.